app/views/registrationamendment/section_list.scala.html

```diff
@@ -19,19 +19,13 @@
 @import views.html.registrationamendment.section
 
 @(sections: Seq[Section], newSections: Option[Seq[Section]] = None)(implicit request: Request[_], m: Messages)
-
     @* new sections *@
-    @newSections.collect { case seq if seq.nonEmpty =>
-        <ul id="new-sections-list" class="grid-layout__column grid-layout__column--2-3 display-block">
-            @seq.map { s =>
-                @submissionSection(s.name, s.status, s.call)
-            }
-        </ul>
-    }
+
+    @allSections = @{sections union newSections.getOrElse(Seq())}
 
     @* existing *@
     <ul id="existing-sections-list" class="grid-layout__column grid-layout__column--2-3 display-block">
-        @sections.map { case Section(name, status, hasChanged, call) =>
-            @section(name, status, hasChanged, call)
+        @allSections.sortBy(section => Messages(s"progress.${section.name}.name")).map { s =>
+            @{ if(sections.contains(s)) section(s.name, s.status, s.hasChanged, s.call) else submissionSection(s.name, s.status, s.call)}
         }
```

Comment:
this will for example put ASP on top, and will mix the order of mandatory and optional sections, is that what we want to achieve?

Reply:
I've removed the ordering of sections - it looks like the default order is the order they're added (hence mandatory sections appearing first). All sections are now displayed in the same list with default ordering, so new sections will be added at the bottom.

Reply:
looks ok 👍